sunpy/map/tests/test_mapbase.py

```diff
@@ -35,35 +36,51 @@ def aia171_test_map():
 
 
 @pytest.fixture
+def heliographic_test_map():
+    return sunpy.map.Map(
+        os.path.join(testpath, 'heliographic_phase_map.fits.gz'))
+
+
+@pytest.fixture
+def heliographic_test_map():
+    return sunpy.map.Map(
+        os.path.join(testpath, 'heliographic_phase_map.fits.gz'))
+
+
+@pytest.fixture
 def aia171_test_map_with_mask(aia171_test_map):
     shape = aia171_test_map.data.shape
     mask = np.zeros_like(aia171_test_map.data, dtype=bool)
-    mask[0:shape[0]//2, 0:shape[1]//2] = True
-    return sunpy.map.Map(np.ma.array(aia171_test_map.data, mask=mask), aia171_test_map.meta)
+    mask[0:shape[0] // 2, 0:shape[1] // 2] = True
+    return sunpy.map.Map(np.ma.array(
+        aia171_test_map.data, mask=mask),
+                         aia171_test_map.meta)
```

Comment:
Is it really needed to break this line into 3? wouldn't be into 2 more readable?
```python
return sunpy.map.Map(np.ma.array(aia171_test_map.data, mask=mask),
                     aia171_test_map.meta)
```